push_notification/subscriber/services/subscriber_service.py

```diff
@@ -0,0 +1,48 @@
+from subscriber.models.subscriber import Subscriber
+
+
+class SubscriberService(object):
+
+    def __init__(self, subscriber, serializer):
+        self.subscriber = subscriber
+        self.serializer = serializer
+
+    def get_subscriber(self, pk=None):
+        "Return subscriber details by given id"
+        if pk is not None:
```

Comment:
1. Is there any use case for default value of `pk` to be None?
2. `pk` is not exactly communicative of which column/key you are referring to
3. instead of a generic variable name like queryset or response, we can rather use `subscriber`
5. We can create an API service layer that communicates with this service layer. the API service layer will communicate with other apps. We can use the serialiser at API service layer.